src/splitInteger.test.js

```diff
@@ -1,17 +1,27 @@
-const splitInteger = require('./splitInteger');
+'use strict';
 
-test(`should split a number into equal parts if a value is divisible by a numberOfParts`, () => {
+const splitInteger = require('./splitInteger');
 
+test(`should split a number into equal parts 
+if a value is divisible by a numberOfParts`, () => {
+  expect(splitInteger(8, 2)).toEqual([4, 4]);
 });
 
-test(`should return a part equals to a value when splitting into 1 part`, () => {
-
+test(`should return a part equals to a value 
+when splitting into 1 part`, () => {
+  expect(splitInteger(10, 1)).toEqual([10]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
+  expect(splitInteger(17, 4)).toEqual([4, 4, 4, 5]);
+});
 
+test('should add zeros if value < numberOfParts', () => {
+  expect(splitInteger(1, 2)).toEqual([0, 1]);
 });
 
 test('should add zeros if value < numberOfParts', () => {
+  const result = splitInteger(3, 4);
 
+  expect(result).toBeInstanceOf(Array);
```

Comment:
the name of this test case does not correspond with the check itself, what are you expecting here? and the name is duplicated twice, the same in the previous test case